cogs/config.py

```diff
@@ -16,3 +16,8 @@
 for i in range(0, 15):
     for j in range(0, 15):
         SCORES.append(f'{i}-{j}')
+
+VALID_SCORES = ('2-0', '2-1', '1-2', '0-2', '1-0', '0-1')
```

Comment:
these constants are being used in multiple files. To avoid duplication and follow DRY patterns, I have moved these to the config file.